app/core/initial_data/UrlType_export.json

```diff
@@ -0,0 +1,57 @@
+[
+    {
+        "id": "1",
+        "name": "Readme",
+        "description": "The GitHub readme.md file for the project"
+    },
+    {
+        "id": "2",
+        "name": "HfLA Website",
+        "description": "The URL of the project's page on the hackforla.org website"
+    },
+    {
+        "id": "3",
+        "name": "GitHub",
+        "description": "Project's primary GitHub repo"
+    },
+    {
+        "id": "4",
+        "name": "Slack",
+        "description": "Project's slack channel"
+    },
+    {
+        "id": "5",
+        "name": "Google Drive",
+        "description": "The url to the project Google Drive"
+    },
+    {
+        "id": "6",
+        "name": "Wiki",
+        "description": "The wiki for the project"
+    },
+    {
+        "id": "7",
+        "name": "Test Site",
+        "description": "Development site used for testing"
+    },
+    {
+        "id": "88",
```

Comment:
I guess the id number was "8"?

Reply:
Yeah, good catch. So this is a consequence of the id being set to 88 in the Excel sheet (as of writing this). You can see that [here](https://docs.google.com/spreadsheets/d/1x_zZ8JLS2hO-zG0jUocOJmX16jh-DF5dccrd_OEGNZ0/edit?gid=329100364#gid=329100364). I can make the update and re-push. Thanks